identity/index.html

```diff
@@ -225,9 +225,7 @@ <h3>MediaDevices.setCaptureHandleConfig()</h3>
               <li>
                 For every capturing application for which the new [=observable=] {{CaptureHandle}}
                 is different than prior to the call to {{MediaDevices/setCaptureHandleConfig}}, an
-                event of type {{CaptureHandleChangeEvent}} must be fired with
-                {{CaptureHandleChangeEvent/captureHandle}} set to the new [=observable=]
-                {{CaptureHandle}}.
+                event named "capturehandlechange" must be fired.
```

Comment:
I believe it should say

```
the user agent MUST queue a task to fire an event with the name "capturehandlechange" at track.
```

Reply:
Track needs to be defined there as well

Reply:
This will be fixed in #56.

Reply:
Should be fixed by #56.

Reply:
To keep things more readable, I will switch from the passive voice ("an event must be fired") to the active voice ("the user agent MUST fire an event").